### PR TITLE
Auth pages

### PR DESCRIPTION
**Pull request by Muzammil-khan710**
added login and signup pages

### CODE REVIEW

File: pages/login.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    <nav class="navbar">
+        <div class="nav-contents">
+          <a class="link-home-page link" href="/index.html"> Mkart </a>
+          <input class="input" type="text" placeholder="Search" />
+  
+          <a href="#"
+            ><img
+              class="icon-navbar"
+              src="https://api.iconify.design/mdi:cart.svg?color=currentColor"
+              alt=""
```
Comment (JuvitaSaini09) [inline pages/login.html]: alt tag is empty

File: pages/login.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+          /></a>
+  
+          <a href="#"
+            ><img
+              class="icon-navbar"
+              src="https://api.iconify.design/ph:heart-fill.svg?color=currentColor"
+              alt=""
```
Comment (JuvitaSaini09) [inline pages/login.html]: here too alt tag is empty

**Review by JuvitaSaini09 — APPROVED** [review]
your code looks good to merge,just add info in alt tag

File: pages/login.html (hunk 2 above)
Comment (Muzammil-khan710) [reply pages/login.html]: updated thanks

**Review by rishabh486 — APPROVED** [review]
well maintained code structure 
good to merge

**Pull request merged**